wrappers/msvcenv.sh

```diff
@@ -34,8 +34,8 @@ SDKLIB="$SDKBASE\\lib\\$SDKVER"
 BINDIR=$BASE_UNIX/vc/tools/msvc/$MSVCVER/bin/Hostx64/$ARCH
 SDKBINDIR=$BASE_UNIX/$SDK_UNIX/bin/$SDKVER/x64
 MSBUILDBINDIR=$BASE_UNIX/MSBuild/Current/Bin/amd64
-export INCLUDE="$MSVCDIR\\include;$SDKINCLUDE\\shared;$SDKINCLUDE\\ucrt;$SDKINCLUDE\\um;$SDKINCLUDE\\winrt"
-export LIB="$MSVCDIR\\lib\\$ARCH;$SDKLIB\\ucrt\\$ARCH;$SDKLIB\\um\\$ARCH"
+export INCLUDE="$MSVCDIR\\include;$SDKINCLUDE\\shared;$SDKINCLUDE\\ucrt;$SDKINCLUDE\\um;$SDKINCLUDE\\winrt;$SDKINCLUDE\\km"
+export LIB="$MSVCDIR\\lib\\$ARCH;$SDKLIB\\ucrt\\$ARCH;$SDKLIB\\um\\$ARCH;$SDKLIB\\km\\$ARCH"
```

Comment:
Curious question; if installing the WDK on Windows, does one get both `um` and `km` directories added to `$INCLUDE` and `$LIB` there as well, in e.g. a developer command prompt?

Reply:
I actually have not checked that, because I've only built drivers via MSBuild, and it does not use these variables. Unfortunately, msvc-wine does not install `vsdevcmd.bat`, so I can not check it here on Linux. I'll look into it when I get to a Windows machine.

Reply:
I've run `vcvars64.bat` from a VC install where the WDK was also installed, and the `\km\` paths were not added to `INCLUDE` and `LIB`. Now I'm not so sure that adding them to default paths was the right thing to do. It could simplify writing plain Makefiles and CMakeLists.txt for the drivers though, without resorting to something like https://github.com/SergiusTheBest/FindWDK.

Reply:
To follow up on this; I think it's probably fine to add them by default here. The majority of users won't install it anyway, so it won't bother them. But for those who do install the WDK, I guess there aren't any conflicts between the sets of headers, so that users would need to worry about which set of headers is included first? If there are no conflicts, this is probably best.

(Otherwise, it'd be really messy for user projects to try to figure out what paths to add, when running within msvc-wine.)

Reply:
As far as I know there are no name conflicts between the `km` and `um` headers and libraries. There's a separate set of standard C library headers for the kernel installed in `km/crt`, but this directory is not added to the default include path.

Reply:
Ok, thanks! I guess that concludes my questions then - I'll go ahead and merge it then!